test/xds/xds_server_rbac_test.go

```diff
@@ -660,6 +691,18 @@ func (s) TestRBACHTTPFilter(t *testing.T) {
 				}
 				// Toggle RBAC back on for next iterations.
 				envconfig.XDSRBAC = true
+
+				if test.wantAuthzOutcomes != nil {
+					if diff := cmp.Diff(lb.authzDecisionStat, test.wantAuthzOutcomes); diff != "" {
+						t.Errorf("Authorization decision do not match\ndiff (-got +want):\n%s", diff)
+					}
+				}
+				if test.eventContent != nil {
+
+					if diff := cmp.Diff(lb.lastEvent, test.eventContent); diff != "" {
+						t.Errorf("Unexpected Message\ndiff (-got + want):\n%s", diff)
```

Comment:
Why is it called "unexpected message"? Maybe something like event content makes more sense?

Reply:
Changed to `Unexpected event`